packages/cms-lib/lib/config.js

```diff
@@ -546,12 +556,10 @@ const updateDefaultAccount = defaultAccount => {
   }
 
   const config = getAndLoadConfigIfNeeded();
-  if (config.defaultAccount) {
-    config.defaultAccount = defaultAccount;
-  }
-  // Keep for backcompat
+  config.defaultAccount = defaultAccount;
+
   if (config.defaultPortal) {
-    config.defaultPortal = defaultAccount;
+    delete config.defaultPortal;
```

Comment:
Since we're deleting, should we tell the user things are changing?

Reply:
Oh yeah good call, will do

Reply:
Where are we copying `defaultPortal` to `defaultAccount`?

Reply:
We aren't copying it over, but this only runs when a new value is being passed in, so a couple lines above we are doing `config.defaultAccount = defaultAccount;` which takes in the arg passed to the function

Reply:
Got it. I didn't read the change with enough context visible.